src/power_grid_model_io/converters/pandapower_converter.py

```diff
@@ -631,34 +644,53 @@ def _create_pgm_input_sym_loads(self):
             data_type=DatasetType.input, component_type=ComponentType.sym_load, shape=3 * n_loads
         )
 
-        const_i_multiplier = (
-            self._get_pp_attr("load", "const_i_percent", expected_type="f8", default=0) * scaling * (1e-2 * 1e6)
-        )
-        const_z_multiplier = (
-            self._get_pp_attr("load", "const_z_percent", expected_type="f8", default=0) * scaling * (1e-2 * 1e6)
-        )
-        const_p_multiplier = (1e6 - const_i_multiplier - const_z_multiplier) * scaling
+        if pp_curr_version <= pp_ref_version:
+            const_i_p_multiplier = (
+                self._get_pp_attr("load", "const_i_percent", expected_type="f8", default=0) * scaling * (1e-2 * 1e6)
+            )
+            const_z_p_multiplier = (
+                self._get_pp_attr("load", "const_z_percent", expected_type="f8", default=0) * scaling * (1e-2 * 1e6)
+            )
+            const_p_multiplier = (1e6 - const_i_p_multiplier - const_z_p_multiplier) * scaling
+            const_q_multiplier = const_p_multiplier
+            const_i_q_multiplier = const_i_p_multiplier
+            const_z_q_multiplier = const_z_p_multiplier
+        else:
+            const_i_p_multiplier = (
+                self._get_pp_attr("load", "const_i_p_percent", expected_type="f8", default=0) * scaling * (1e-2 * 1e6)
+            )
+            const_z_p_multiplier = (
+                self._get_pp_attr("load", "const_z_p_percent", expected_type="f8", default=0) * scaling * (1e-2 * 1e6)
+            )
+            const_p_multiplier = (1e6 - const_i_p_multiplier - const_z_p_multiplier) * scaling
+            const_i_q_multiplier = (
+                self._get_pp_attr("load", "const_i_q_percent", expected_type="f8", default=0) * scaling * (1e-2 * 1e6)
+            )
+            const_z_q_multiplier = (
+                self._get_pp_attr("load", "const_z_q_percent", expected_type="f8", default=0) * scaling * (1e-2 * 1e6)
+            )
+            const_q_multiplier = (1e6 - const_i_q_multiplier - const_z_q_multiplier) * scaling
```

Comment:
cfr. my other comment about pandapower version availability: this could be resolved either using a `try/except` or using some container contains checks